pkg/vpa/service.go

```diff
@@ -149,6 +149,44 @@ func (c *Service) CreateTortoiseUpdaterVPA(ctx context.Context, tortoise *autosc
 	return vpa, tortoise, nil
 }
 
+func (c *Service) UpdateVPACRP(ctx context.Context, tortoise *autoscalingv1beta3.Tortoise) (*v1.VerticalPodAutoscaler, error) {
+	retVPA := &v1.VerticalPodAutoscaler{}
+
+	// we only want to record metric once in every reconcile loop.
+	//metricsRecorded := false
+	updateFn := func() error {
+		vpa, err := c.GetTortoiseUpdaterVPA(ctx, tortoise)
+		if err != nil {
+			return fmt.Errorf("get tortoise VPA: %w", err)
+		}
```

Comment:
Instead of getting VPA via `GetTortoiseUpdaterVPA`, I think we can just add vpa in the function parameter and pass it from `Reconcile` function.

Reply:
It's still left.